src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorInitializeEndpoint.cs

```diff
@@ -4,14 +4,20 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
+using Microsoft.AspNetCore.Razor.Telemetry;
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.CodeAnalysis.Razor.Workspaces;
 using Microsoft.CommonLanguageServerProtocol.Framework;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer;
 
 [RazorLanguageServerEndpoint(Methods.InitializeName)]
-internal class RazorInitializeEndpoint : IRazorDocumentlessRequestHandler<InitializeParams, InitializeResult>
+internal class RazorInitializeEndpoint(
+        LanguageServerFeatureOptions languageServerFeatureOptions) : IRazorDocumentlessRequestHandler<InitializeParams, InitializeResult>
 {
+    private readonly LanguageServerFeatureOptions _languageServerFeatureOptions = languageServerFeatureOptions;
```

Comment:
Nit: I've been changing these to `options` as I go because I feel like `languageServerFeatureOptions` is overly specific and a bit clunky.